<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FastCmd</class>
 <widget class="QDialog" name="FastCmd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>466</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(19, 156, 106);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(207, 220, 224);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 3</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 4</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Tab 5</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>计算校验</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>请输入16进制数据流：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTextEdit" name="textEdit_data"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QRadioButton" name="radioButton_XOR">
              <property name="text">
               <string>异或</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_Sum">
              <property name="text">
               <string>累加和</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_CRC">
              <property name="text">
               <string>CRC</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_CheckSum">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>计算结果：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_CheckSum">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
